### PR TITLE
Add localization support to Text component

### PR DESCRIPTION
**Pull request by PavelHolec**
The components update will follow in #780

### CODE REVIEW

File: Sources/Orbit/Components/Text.swift [hunk 1]
```diff
@@ -304,18 +345,83 @@ public struct Text: View, FormattedTextBuildable, PotentiallyEmptyView {
...
+    @_semantics("swiftui.init_with_localization")
+    init(
+        _ key: StaticString,
+        value: LocalizedStringKey,
+        tableName: String? = nil,
+        bundle: Bundle? = nil,
+        comment: StaticString? = nil
+    ) {
+        self.localization = .key(value, bundle: bundle, tableName: tableName, explicitKey: key.description)
+        self.verbatimContent = value.key ?? ""
```
Comment (sjavora) [inline Sources/Orbit/Components/Text.swift:370]: Shouldn't this fall back to `key`?

File: Sources/Orbit/Support/Text/LocalizedStringKey.swift [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                forResource: locale.identifier.split(separator: "_").map(String.init).first ?? locale.identifier, 
+                ofType: "lproj"
+            )
+        return (bundlePath.map(Bundle.init(path:)) ?? self) ?? self
+    }
+}
+
+public extension LocalizedStringKey {
+
+    private typealias FArgs = (CVarArg, Formatter?)
```
Comment (sjavora) [inline Sources/Orbit/Support/Text/LocalizedStringKey.swift:17]: `FormatterArguments`? I really can't tell what this short name is supposed to be... 😅

Comment (PavelHolec) [reply Sources/Orbit/Support/Text/LocalizedStringKey.swift]: Renamed to `FormattedVarArgs`

File: Sources/Orbit/Components/Text.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/Text.swift]: Updated

**Review by sjavora — APPROVED** [review]

**Pull request merged**